Search.jsp

```diff
@@ -158,6 +162,7 @@ select * from article where title like '%<%=request.getAttribute("search2") %>%'
 <c:forEach items="${result.rows}" var="row">
 <c:set value="${ search2}" var="search2"></c:set>
 <tr>
+<td><c:out value="${ row.subject}"></c:out></td>

```

Comment:
Should be `${row.subject}`, no space between `${` and `row`.

**Hui**

Reply:
I believe it is not going to affect anything.

Reply:
> I believe it is not going to affect anything.

@mjhmohamed 

Functionally speaking, yes.  But it violates Code Style.   

**Hui**

Reply:
OK. I'll try to not violate the Code Style.